thedmstoolkit/character_generation.py

```diff
@@ -0,0 +1,538 @@
+            "Shifter",
+            "Tabaxi",
+            "Tiefling",
+            "Triton",
+            "Tortle",
+            "Vedalken",
+            "Warforged",
+            "Locathah",
+            "Bugbear",
+            "Goblin",
+            "Hobgoblin",
+            "Kobold",
+            "Lizardfolk",
+            "Orc",
+            "Dwarf",
+            "Elf",
+            "Gnome",
+            "Half-Elf",
+            "Half-Orc",
+            "Halfling",
+            "Human",
+        ],
+    }
+    CLASS_DICT: dict[str, list] = {
+        "Martial": ["Fighter", "Monk", "Ranger", "Rogue"],
+        "Divine": ["Cleric", "Paladin", "Warlock"],
+        "Magic": ["Artificer", "Bard", "Druid", "Sorcerer", "Wizard"],
+        "All": [
+            "Fighter",
+            "Monk",
+            "Ranger",
+            "Rogue",
+            "Cleric",
+            "Paladin",
+            "Warlock",
+            "Artificer",
+            "Bard",
+            "Druid",
+            "Sorcerer",
+            "Wizard",
+        ],
+    }
+    ALIGNMENT_DICT: dict[str, list] = {
+        "Good": ["Lawful Good", "Neutral Good", "Chaotic Good"],
+        "Neutral": ["Lawful Neutral", "True Neutral", "Chaotic Neutral"],
+        "Evil": ["Lawful Evil", "Neutral Evil", "Chaotic Evil"],
+        "All": [
+            "Lawful Good",
+            "Neutral Good",
+            "Chaotic Good",
+            "Lawful Neutral",
+            "True Neutral",
+            "Chaotic Neutral",
+            "Lawful Evil",
+            "Neutral Evil",
+            "Chaotic Evil",
+        ],
+    }
+    BACKGROUND_LIST = [
+        "Acolyte",
+        "Athlete",
+        "Barbarian Tribe Member",
+        "Charlatan",
+        "City Watch",
+        "Criminal",
+        "Clan Crafter",
+        "Entertainer",
+        "Faction Agent",
+        "Far Traveler",
+        "Fisher",
+        "Folk Hero",
+        "Gladiator",
+        "Hermit",
+        "Knight",
+        "Noble",
+        "Outlander",
+        "Pirate",
+        "Sage",
+        "Sailor",
+        "Soldier",
+        "Urchin",
+    ]
+    GENERATOR_LIST: list[str] = ["Stats", "Race", "Class", "Alignment", "Background"]
+    STANDARD_ARRAY: list[int] = [15, 14, 13, 12, 10, 8]
+
+    @staticmethod
+    def three_d_six(_low, _high) -> int:
+        """Returns the total od 3d6
+
+        Args:
+            _low (_type_): Ignored
+            _high (_type_): Ignored
+
+        Returns:
+            int: Returns the total of 3d6
+        """
+        return random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6)
+
+    Generators: dict[str, Generator] = {}
+    # Unlimited Generators have a minimum range of [1,max_int]
+    UnlimitedGenerators: dict[str, Generator] = {"random": random.randint}
+    # Limited Generators have a max range of [1,20], and therefore are not suitable for use in generating race, class, etc
+    LimitedGenerators: dict[str, Generator] = {
+        # "3d6": three_d_six,
```

Comment:
is there a problem with 3d6 right now?

Reply:
You are not allowed to call static methods out of the dictionary, so it's literally unusable

Reply:
Does it have to be `Character_Generator.three_d_six` in the dict?